phoenix-pherf/src/main/java/org/apache/phoenix/pherf/jmx/MonitorManager.java

```diff
@@ -135,10 +154,7 @@ public synchronized boolean shouldStop() {
         return shouldStop;
     }
 
-    public synchronized void stop() {
-        this.shouldStop = true;
-    }
-
+    @SuppressWarnings("unused")
```

Comment:
Why is this annotation needed here? If the method if not being used, then we should just remove it?